recipes/nvitop-exporter/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nvitop-exporter" %}
+{% set version = "1.4.0" %}
+{% set sha256 = "70b1793f4c23b2c703216422816e1916a6162d78b8035ff9183301d5d098e86d" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name|replace("-", "_") }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Package `windows-curses` is not available yet
+  skip: true  # [win]
```

Comment:
Looks like this one can be noarch, no?

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to 
build and the package will be ready when the missing dependency is available on Windows.

Reply:
> Looks like this one can be noarch, no?

Yes. It is a noarch package that only contains pure Python code.

> It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

The CI will run on linux, osx, and windows. The windows CI will fail due to missing dependency.